server/autotest_server/__init__.py

```diff
@@ -139,16 +139,16 @@ def _get_feedback(test_data, tests_path, test_id):
         feedback_path = os.path.join(tests_path, feedback_file)
         if os.path.isfile(feedback_path):
             with open(feedback_path, "rb") as f:
-                now = int(time.time())
-                key = f"autotest:feedback_file:{test_id}:{now}"
                 conn = redis_connection()
+                id_ = conn.incr("autotest:feedbacks_id")
```

Comment:
@pretendWhale sorry for the delay in looking at this! I think a better name would be `feedback_files_id` to match the standard naming convention